### PR TITLE
New UI for Review Submissions page

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes projecthydra-labs/hyku#682
screen shot 2017-02-01 at 7 57 14 pm

### CODE REVIEW

File: app/controllers/hyrax/admin/workflows_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  class Admin::WorkflowsController < ApplicationController
+    before_action :ensure_admin!
+    layout 'admin'
+
+    def index
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.toolbar.admin.menu'), hyrax.admin_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.tasks'), '#'
+      add_breadcrumb t(:'hyrax.admin.sidebar.workflow_review'), request.path
+      @status_list = Hyrax::Workflow::StatusListService.new(self, "-workflow_state_name_ssim:#{deposited_workflow_state_name}")
```
Comment (mjgiarlo) [inline app/controllers/hyrax/admin/workflows_controller.rb]: Should we put `workflow_state_name_ssim` behind a new private method that uses Solrizer to construct the key name, or meh?

File: app/services/hyrax/workflow/status_list_service.rb [hunk 2]
```diff
@@ -35,11 +38,12 @@ def search_solr
...
-          WorkRelation.new.search_with_conditions(
-            { actionable_workflow_roles_ssim: actionable_roles },
-            fl: 'id title_tesim has_model_ssim, workflow_state_name_ssim',
-            rows: 1000
-          )
+          WorkRelation.new.search_with_conditions(query(actionable_roles), rows: 1000)
+        end
+
+        def query(actionable_roles)
+          ["{!terms f=actionable_workflow_roles_ssim}#{actionable_roles.join(',')}",
```
Comment (mjgiarlo) [inline app/services/hyrax/workflow/status_list_service.rb]: Should we put this solr key behind a method for easier refactoring/overriding?

File: app/services/hyrax/workflow/status_list_service.rb [hunk 3]
```diff
@@ -1,9 +1,12 @@
 module Hyrax
   module Workflow
+    # Finds a list of works that we can perform a workflow action on
     class StatusListService
       # @param context [#current_user, #logger]
-      def initialize(context)
+      # @param filter_condition [String] a solr filter
+      def initialize(context, filter_condition)
         @context = context
+        @filter_condition = filter_condition
```
Comment (mjgiarlo) [inline app/services/hyrax/workflow/status_list_service.rb]: Should this be added to the `attr_reader` list?

**Comment by mjgiarlo** [comment]
Would you mind adding a screenshot to this PR, @jcoyne?

File: app/controllers/hyrax/admin/workflows_controller.rb (hunk 1 above)
Comment (jcoyne) [inline app/controllers/hyrax/admin/workflows_controller.rb]: YAGNI

File: app/services/hyrax/workflow/status_list_service.rb (hunk 3 above)
Comment (jcoyne) [inline app/services/hyrax/workflow/status_list_service.rb]: Why?

File: app/services/hyrax/workflow/status_list_service.rb (hunk 2 above)
Comment (jcoyne) [inline app/services/hyrax/workflow/status_list_service.rb]: YAGNI

**Pull request merged**